fiatconnect-api.md

```diff
@@ -2227,12 +2230,67 @@ A KYC schema containing personal data about a user, as well as documents such as
 	},
 	phoneNumber: `string`,
 	selfieDocument: `string`,
-	identificationDocument: `string`
+  identificationDocument: `string`
 }
 ```
 
 The `selfieDocument` and `identificationDocument` fields should be base64 encoded binary blobs representing images.
 
+#### 9.3.1.2. `PersonalDataAndDocumentsDetailed`
+A more detailed Schema allowing the user to provide more information to work with CICO providers that require it.
+
+```
+{
+	firstName: `string`,
+	middleName?: `string`,
+	lastName: `string`,
+	dateOfBirth: {
+		day: `string`,
+		month: `string`,
+		year: `string`
+	},
+	address: {
+		address1: `string`,
+		address2?: `string`,
+		isoCountryCode: `string`,
+		isoRegionCode: `string`,
+		city: `string`,
+		postalCode?: `string`
+	},
+	phoneNumber: `string`,
+  email?: `string`,
+	selfieDocument: `string`,
+  identificationDocumentType: `IdentificationDocumentTypeEnum`,
+	identificationDocumentFront: `string`,
+  identificationDocumentBack?: `string` 
+}
+```
+
+The `phoneNumber` field is REQUIRED and MUST follow the formating of the [E.164 international standard](https://en.wikipedia.org/wiki/E.164). It MUST contain all parts of the phone number INCLUDING the Country Code.
+
+The `email` field MUST be a [valid email](https://en.wikipedia.org/wiki/Email_address#Syntax).
+
+The `identificationDocumentType` field must be selected from `IdentificationDocumentTypeEnum`, and is used to represent the kind of document being submitted.
+
+The `selfieDocument`, `identificationDocumentFront` and `identificationDocumentFront` fields MUST be base64 encoded binary blobs representing images.
+
+The `identificationDocumentBack` field is REQUIRED if the `identificationDocumentType` is `IDC` or `DL`. Otherwise, it is optional.
```

Comment:
`OPTIONAL` is a BCP 14 keyword.
```suggestion
The `identificationDocumentBack` field is REQUIRED if the `identificationDocumentType` is `IDC` or `DL`. Otherwise, it is OPTIONAL.
```

Reply:
Corrected